EIC-EA/EIC-community-D8
# EICNET-2061: Unsubscribe/On-Off for notification settings

## Important note
Currently only the following subscribtion messages are fully implemented (keep this in mind while testing the feature):
- SUBSCRIPTION :: MT18 :: New group content published
- SUBSCRIPTION :: MT16 :: New comment (reply) after comment
- SUBSCRIPTION :: MT19 :: New comment on content
- SUBSCRIPTION :: MT20 :: Group content updated
- SUBSCRIPTION :: MT23 :: Content published about Terms of Interest

## To Test

- [x] Go to /user/{user}/settings **_(be sure to have a filled profile before)_**
- [x] Set to "off" your interest and comment settings. Set to "on" some of your group & event settings while logged in as TU
- [x] Log in with another user and perform following actions:
 1. Create a news with one of the topic the previously used TU account has on its profile
 2. Create a discussion in one of the groups where notifications are allowed
 3. Comment as this second(you can also use a third) account on the news and the discussion
- [x] Run cron
- [x] Expected results:
 1. Message MT23 shouldn't be sent to the TU following the topic
 2. Message M18 should be sent to TU
 3. Message MT19 shouldn't be sent to TU for the news but should be sent for the discussion
- [x] (Optional) You can also try updating a content within a followed group (be sure that 'Send notification' is checked) and replying to a comment. Depending on your settings messages MT20 and MT16 should/shouldn't be sent.
